File: DailyFlash Solutions/DailyFlash Solutions/36-DailyFlash_Solutions/24_Feb_Solution_Three/Python/prog5.py
import array
num = int(input("Enter num : "))
arr = array.array('i', [0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
while(num != 0):
    arr[num % 10] += 1
    num = int(num / 10)
for i in range(10):
    if(arr[i] != 0):
        print("Freqn of", i, "is", arr[i])

File: DailyFlash Solutions/DailyFlash Solutions/36-DailyFlash_Solutions/24_Feb_Solution_Three/C++/prog5.cpp
#include<iostream>

int main() {
	int num, sum = 0, cnt = 0;
	std::cout << "Enter num : ";
	std::cin >> num;
	int temp = num;
	int arr[10] = {0};
	while(num != 0) {
		arr[num % 10] += 1;
		num /= 10;
	}	
	for(int i = 0; i < 10; i++) {
		if(arr[i] != 0) {
			std::cout << "Freqn of " << i << " is " << arr[i] << "\n";	
		}
	}
}
